techlibs/gatemate/brams.txt

```diff
@@ -12,7 +12,7 @@ ram block $__CC_BRAM_TDP_ {
 	option "MODE" "CASCADE" {
 		abits 16;
 		# hack to enforce same INIT layout as in the other modes
-		widths 1 2 5 per_port;
+		widths 1 per_port;
```

Comment:
please keep this; removing the extra widths will change the initial data layout (the pass will no longer emit the parity bits) and break functionality

Reply:
(note that `2 5` can never actually be picked for a port, due to the `width mix 1` override later on